changes/issue-4361-mail-change-should-error

```diff
@@ -0,0 +1 @@
+* Check if the email being changed in a user already exists before sending the request
```

Comment:
nitpick, awkward grammer?
```suggestion
* Check if the email being changed for a user already exists before sending the request
```

Reply:
I guess my mind was thinking "_in_ the user struct".